server/routes/healthy.js

```diff
@@ -1,13 +1,26 @@
 const dbService = require('../services/database-service')
+const DbVersion = require('../dbversion')
+const dbVersion = new DbVersion()
 
 module.exports = {
   method: 'GET',
   path: '/healthy',
   options: {
     handler: async (request, h) => {
-      if (await dbService.isConnected()) {
+      const failures = []
```

Comment:
Would it be useful to encapsulate this logic in another file to keep the route cleaner?

Reply:
Yes, good idea.

Reply:
Yes, good suggestion. I've moved it out into a separate file.